packages/dialtone-vue2/components/popover/popover.vue

```diff
@@ -917,6 +962,56 @@ export default {
         onShow: this.onShow,
       });
     },
+
+    //  ============================================================================
+    //  $ HOVERCARD
+    //  ----------------------------------------------------------------------------
+
+    setInTimer () {
+      this.inTimer = setTimeout(() => {
+        this.isOpen = true;
+      }, TOOLTIP_DELAY_MS);
+    },
+
+    setOutTimer () {
+      this.outTimer = setTimeout(() => {
+        this.isOpen = false;
+      }, TOOLTIP_DELAY_MS);
+    },
+
+    onEnterAnchor () {
+      if (!this.hovercard) return;
```

Comment:
I would suggest to have computed property `isHovercard` to improve readability, instead to use this.hovercard